src/components/vr_module/include/vr_module/socket_channel.h

```diff
@@ -47,6 +47,7 @@ namespace vr_module {
 class SocketChannel : public Channel {
  public:
   SocketChannel();
+  explicit SocketChannel(net::ConnectedSocket *socket);
```

Comment:
Why? I have heard that it is good practice to use star near variable. Google style allows to use star that way.